changelog/unreleased/4567

```diff
@@ -0,0 +1,6 @@
+Fix: Add bottom margin for used quota in account dialog
```

Comment:
The tag should be `Bugfix` instead of `Fix`, as mentioned in the Calens [README.md](https://github.com/owncloud/android/blob/master/changelog/README.md)